### PR TITLE
trace statements in delete and list tasks

### PR DESCRIPTION
**Pull request by teddyward**
unsure if included too many or not enough or just the right amount

### CODE REVIEW

File: app/exec/build-tasks-delete.ts [hunk 1]
```diff
@@ -26,20 +27,27 @@ export var hideBanner: boolean = false;
 
 export class BuildTaskDelete extends cmdm.TfCommand {
     public exec(args: string[], options: cm.IOptions): any {
+        trace("User requested to delete a build task.");
```
Comment (bryanmacfarlane) [inline app/exec/build-tasks-delete.ts]: let's start a pattern of build-tasks-delete.exec

File: app/exec/build-tasks-delete.ts [hunk 2]
```diff
@@ -26,20 +27,27 @@ export var hideBanner: boolean = false;
...
         var agentapi = this.getWebApi().getTaskAgentApi(this.connection.accountUrl);
 
+        trace("Searching for tasks with id: " + taskId);
         agentapi.getTaskContent(taskId, "", (err, statusCode, tasks) => {
+            trace("Found " + tasks.length + " tasks with provided id.")
             if (tasks.length > 0) {
+                trace("Deleting task(s)...");
                 agentapi.deleteTaskDefinition(taskId, (err, statusCode) => {
                     if (err) {
+                        trace("Call to TaskAgentApi.deleteTaskDefinition failed. Message: " + err.message);
```
Comment (bryanmacfarlane) [inline app/exec/build-tasks-delete.ts]: is there also an http code on this?  I think so

File: app/exec/build-tasks-delete.ts [hunk 3]
```diff
@@ -26,20 +27,27 @@ export var hideBanner: boolean = false;
...
                         err.statusCode = statusCode;
                         deferred.reject(err);
                     }
                     else {
+                        trace("Successfully deleted task(s).")
```
Comment (bryanmacfarlane) [inline app/exec/build-tasks-delete.ts]: terse success is fine

File: app/exec/build-tasks-delete.ts [hunk 4]
```diff
@@ -47,6 +55,7 @@ export class BuildTaskDelete extends cmdm.TfCommand {
                 });
             }
             else {
+                trace("No task found with provided id: " + taskId);
```
Comment (bryanmacfarlane) [inline app/exec/build-tasks-delete.ts]: same - terse 'no task' is fine

File: app/exec/build-tasks-list.ts [hunk 5]
```diff
@@ -25,19 +26,27 @@ export var hideBanner: boolean = false;
```
Comment (bryanmacfarlane) [inline app/exec/build-tasks-list.ts]: same feedback as file above

**Comment by teddyward** [comment]
see https://github.com/Microsoft/tfs-cli/pull/7 for better commit history

**Pull request closed without merging**